webapp/src/containers/components/modals/deleteModal.tsx

```diff
@@ -20,7 +17,7 @@ function DeleteModal(props: Props) {
 
     const handleClose = () => {
         setShow(false);
-        props.setVisibility(false);
+        props.setVis(false);
```

Comment:
Why this change

Reply:
fixed the variable name to setVisibility